### PR TITLE
Update License and attributions

### PR DESCRIPTION
**Pull request by pbernasconi**
@erimag  –the Plaid team would like to thank you for your contributions to the plaid-java library.

We are simply correcting the copyright attribution to reflect that ownership. Thanks again, so much, for your contributions! We have attributed your name in the README, as noted below.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -69,3 +69,6 @@ There are two main dependencies (of course automatically managed by Maven):
 
 The project also has a test scope dependency on [Wiremock](http://wiremock.org), which is really useful in order to avoid hitting the sandbox API all the time.
 
+
+### Contributors
+- @erimag (Qapital Insight AB)
```
Comment (davidchambers) [inline README.md]: Let's have the username link to Erik's GitHub profile. :)

File: README.md [hunk 2]
```diff
@@ -69,3 +69,6 @@ There are two main dependencies (of course automatically managed by Maven):
 
 The project also has a test scope dependency on [Wiremock](http://wiremock.org), which is really useful in order to avoid hitting the sandbox API all the time.
 
+
```
Comment (davidchambers) [inline README.md]: :space_invader:

File: README.md (hunk 1 above)
Comment (pbernasconi) [inline README.md]: ⚡

File: README.md (hunk 2 above)
Comment (pbernasconi) [inline README.md]: ⚡

File: README.md [hunk 3]
```diff
@@ -69,3 +69,5 @@ There are two main dependencies (of course automatically managed by Maven):
 
 The project also has a test scope dependency on [Wiremock](http://wiremock.org), which is really useful in order to avoid hitting the sandbox API all the time.
 
+### Contributors
+- [@erimag](https://github.com/erimag) (Qapital Insight AB)
```
Comment (davidchambers) [inline README.md]: Nit: Let's include an empty line after the heading, to respect the document's formatting.

File: LICENSE [hunk 4]
```diff
@@ -1,6 +1,6 @@
-The MIT License (MIT)
+The MIT License
 
-Copyright (c) 2014 Qapital Insight AB
+Copyright (c) 2014-present Plaid Technologies, Inc. <support@plaid.com>
```
Comment (davidchambers) [inline LICENSE]: `s/2014-present/2016/`

Comment (pbernasconi) [inline LICENSE]: ⚡

File: README.md (hunk 3 above)
Comment (pbernasconi) [inline README.md]: ⚡

**Comment by erimag** [comment]
Sure, happy to merge if you're satisfied with the changes.

**Comment by davidchambers** [comment]
LGTM

**Pull request merged**